tests/operator/operator.go

```diff
@@ -582,8 +582,10 @@ spec:
 				cdi, err := virtClient.CdiClient().CdiV1beta1().CDIs().Get(context.Background(), originalCDI.Name, metav1.GetOptions{})
 				Expect(err).ToNot(HaveOccurred())
 				if !equality.Semantic.DeepEqual(cdi.Spec, originalCDI.Spec) {
-					cdi.Spec = originalCDI.Spec
-					_, err := virtClient.CdiClient().CdiV1beta1().CDIs().Update(context.Background(), cdi, metav1.UpdateOptions{})
+					patchPayload, err := patch.New(patch.WithReplace("/spec", originalCDI.Spec)).GeneratePayload()
+					Expect(err).ToNot(HaveOccurred())
+
+					_, err = virtClient.CdiClient().CdiV1beta1().CDIs().Patch(context.Background(), cdi.Name, types.JSONPatchType, patchPayload, metav1.PatchOptions{})
 					Expect(err).ToNot(HaveOccurred())
 				}
```

Comment:
tbh I am not sure this is needed any longer - I don't see any tests changing the CDI spec.
it's possible we don't need the w/a from this old commit - https://github.com/kubevirt/kubevirt/commit/cfb9cbe96b4198d9290477c35e576e9c766c2710#diff-e3e6528c8b5d4860f4fd89d38f0e0babb9cc0e9147c0b0b28be532e5e692c182R824

Reply:
I noticed that too and I agree with you, but I thought maybe in the future we might introduce tests that will require some changes to the cdi resource and it would be easier to maintain

Reply:
@dasionov I wouldn't reply on that. When these tests would be introduced we will ask the authors to add the relevant clean-up code. But if currently this code isn't needed, we can drop it.

Reply:
done